string-manipulation-1/src/reverse-word.js

```diff
@@ -0,0 +1,38 @@
+/* exported reverseWord */
+function reverseWord(word) {
+  var reverseArr = [];
+  var reverseOutput = [];
+  var finalOutput = [];
+  var finalWord = '';
+  for (var i = 0; i < word.length; i++) {
+    reverseArr.push(word[i]);
+  }
+  for (var j = reverseArr.length; j >= 0; j--) {
+    reverseOutput.push(reverseArr[j]);
+  }
```

Comment:
Why not just do this loop over `word` and save the need for the loop above? Also, if you concatenate to an output rather than use an array, you can avoid the need for the loop that `join` is doing for you.